CMakeLists.txt

```diff
@@ -20,10 +20,10 @@ find_package(re2 CONFIG REQUIRED)
 find_package(c-ares CONFIG REQUIRED)
 find_package(absl CONFIG REQUIRED)
 
-set(HAPI_VERSION_TAG "v0.35.0" CACHE STRING "Use the configured version tag for the Hedera API protobufs")
+set(HAPI_VERSION_TAG "v0.34.0" CACHE STRING "Use the configured version tag for the Hedera API protobufs" FORCE)
```

Comment:
Shouldn't it be `v0.36.0` over here?

Reply:
Yes, that'll be grabbed by the other PR, I'll revert this change